packages/react-components/package.json

```diff
@@ -79,7 +79,7 @@
     "karma": "^6.3.14",
     "karma-chrome-launcher": "^3.1.0",
     "karma-mocha": "^2.0.1",
-    "karma-webpack": "^4.0.2",
+    "karma-webpack": "^5.0.0",
```

Comment:
Did the `dependencies` addition in the root `package.json` cause this to error?

Reply:
That's correct upgrading to webpack 5 in the root caused karma to fail thus causing me to upgrade to karma 5 which supports webpack 5